packages/in_app_purchase/example/android/app/build.gradle

```diff
@@ -27,6 +27,8 @@ project.ext {
     KEYSTORE_STORE_PASSWORD = keystoreProperties['storePassword']
     KEYSTORE_KEY_ALIAS = keystoreProperties['keyAlias']
     KEYSTORE_KEY_PASSWORD = keystoreProperties['keyPassword']
+    VERSION_CODE = 1
```

Comment:
It looks ok to me, but I don't have enough understanding of gradle to review gradle in a high degree of confidence. If you are not confident of this part, please find someone else to take a look.